test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -385,7 +402,8 @@ private static void createUI(String title, String instructions,
         frame.add(createInstructionUIPanel(instructions,
                                            testTimeOut,
                                            rows, columns,
-                                           enableScreenCapture),
+                                           enableScreenCapture,
+                                           false, 0),
```

Comment:
It is intentionally added only to the builder to avoid excessive changes to all constructors.

It seems that the builder produces much cleaner code, so we probably shouldn't spend our time to changing a bunch of `PassFailJFrame` constructors, and at some point we can make the builder the only option available.

Reply:
If we remove the constructors, the builder object can be passed thus eliminating the need to add more parameters. So, it's a good thing to do.